app/controllers/organizations_controller.rb

```diff
@@ -135,9 +135,10 @@ def set_users_github_organizations
   # if so add the user to the corresponding classroom automatically.
   def add_current_user_to_organizations
     @users_github_organizations.each do |organization|
-      classroom = organization[:classroom]
-      if classroom.present? && !classroom.users.include?(current_user)
-        create_user_organization_access(classroom)
+      classrooms = Organization.where(github_id: organization[:github_id])
```

Comment:
Would classrooms be a list of organizations?

Reply:
Yea, they are organizations, but you have a good point this can be confusing. I'm going to rename it to organizations just so it's more clear